spec/components/requestBodies/PatchOrganization.yaml

```
@@ -1,16 +1,10 @@
 content:
   application/json:
     schema:
-      required:
-        - name
       properties:
         name:
-          description: The organization name
+          description: The organization name.  It must be globally unique within Rebilly.
```

```suggestion
          description: The organization name. It must be globally unique within Rebilly.
```
Same in few places in this PR with the same sentence